frontend/src/Home.js

```diff
@@ -15,10 +16,13 @@ export default function Home() {
 
   return (
     <div className="home-content">
-      <h1>Welcome to GreySwan!</h1>
-      <p className="message">
+      <Typography variant="h2" style={{ margin: '30px'}}>
+        Welcome to GreySwan!  
+      </Typography>
+      <img src={require('./greyswan.png')} alt="Grey swan" height="100" width="100" style={{ marginLeft: '10px'}} />
```

Comment:
v nit: make image directory a constant?
so easily changeable in future

Reply:
oops you can't do this with `require`... but good point :')

Reply:
aww, i wonder why though hmm

Reply:
if you wanted to make it a constant that's changeable, i think you can do the following:

```
export const IMAGE_FILE = require('./images/greyswan.png')
```

and then import the IMAGE_FILE from this or something and refer to it as the IMAGE_FILE

Reply:
thank you!!